ereol_inspiration/ereol_inspiration.features.field_instance.inc

```diff
@@ -317,7 +317,7 @@ function ereol_inspiration_field_default_field_instances() {
   t('Categories');
   t('Category query');
   t('Elementer');
```

Comment:
Should 'Elementer' not be listed as 'Elements' as well?

Reply:
Yes, but that error has not been introduced in this pull request (cf. https://github.com/eReolen/ereolen/blame/0ef35d53b2ccf131eb587429728ff0b6f8bf5afb/ereol_inspiration/ereol_inspiration.features.field_instance.inc#L319).